tests/test_and.rs

```diff
@@ -13,69 +13,69 @@ Included cases:
 - Set Zero flag if result is zero
 - Set Negative flag if result is negative
 
-All tests also check for unexpected changes in untargeted lo registers and condition flags
+All tests also check for unexpected changes in registers and condition flags
 TODO: Add tests where untargeted values are pre-set and check if incorrectly reset?
 
 The reference for these tests is currently official documentations and a QEMU-based VM
 TODO: Test against a hardware Cortex-M0 to make sure it's actually up to spec?
 
 */
 
-// Test if AND(... 0000 1111, ... 1010 1010) = ... 0000 1010
+// Test if operation calculate correctly for two register values
 #[test]
 pub fn test_and_register(){
     let mut vm = create_vm_from_asm("
-        movs r0, #0x00_00_00_0F
-        movs r1, #0x00_00_00_AA
+        movs r0,            #0x0000000F
```

Comment:
Just for reference here, it's not strictly necessary to use 0x0000000F instead of just 0x0F. I'd leave that up to style/ease of writing the code preference. However, do always round to the nearest byte for the 0s. For example don't do 0xF and prefer 0x0F, for 0xA03, prefer 0x0A03 etc.

Reply:
Fair enough, 95% of constants will probably fit in the lowest byte either way